playbooks/files/rax-maas/plugins/maas_common.py

```diff
@@ -626,6 +660,12 @@ def get_auth_details(openrc_file=OPENRC):
         if auth_details[key] is None:
             status_err('%s not set' % key, m_name='maas_keystone')
 
+    # NOTE(cloudnull): The password variable maybe double quoted, to
+    #                  fix this we strip away any extra quotes in
+    #                  the variable.
+    pw = auth_details['OS_PASSWORD'].strip('"').strip("'")
```

Comment:
what if the password starts or ends with " or '?

Reply:
perhaps check for it on both ends to reduce the likelihood of this

Reply:
Done